docs/installation.adoc

```diff
@@ -109,7 +109,7 @@ $ echo "deb https://dl.bintray.com/odo/odo-deb-releases stretch main" | sudo tee
 ==== Tarball installation
 
 ----
-# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
+# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64.tar.gz | tar -xvzf odo-linux-amd64.tar.gz -C /usr/local/bin; chmod +x /usr/local/bin/odo'
```

Comment:
@amitkrout this change

Reply:
https://github.com/openshift/odo/pull/2009#discussion_r316532573